crawl-ref/source/spl-cast.cc

```diff
@@ -920,6 +920,10 @@ static void _spellcasting_god_conduct(spell_type spell)
     if (is_corpse_violating_spell(spell))
         did_god_conduct(DID_CORPSE_VIOLATION, conduct_level);
 
+    // need to handle ignite_poison as a DID_FIRE
```

Comment:
reviewing this, I think I should use a comment similar to SPELL_SWIFTNESS below.
// not is_fiery_spell since the other ones handle the conduct themselves